migtests/tests/pg/basic-live-test/snapshot_schema.sql

```diff
@@ -0,0 +1,9 @@
+create table x(id int primary key,id2 int);
+ALTER TABLE user_table REPLICA IDENTITY FULL;
+
+CREATE TABLE user_table (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) UNIQUE,
+    status VARCHAR(50) DEFAULT 'active'
+);
+ALTER TABLE user_table REPLICA IDENTITY FULL; 
```

Comment:
These alter stmts are not required right as we have that `set_replica_identity()`?